OHCRN/keycloak-theme
# #899: style "verify email" email for keycloak

TODO: final text

---

This PR creates a styled template with i18n for Keycloak emails.

- Add template, broken down into components
- Add English & French text
- Add "verify your email" email

Testing instructions:
- Copy the entire theme folder into Keycloak: `docker cp ./keycloak-theme KEYCLOAK_CONTAINER_ID:/opt/bitnami/keycloak/themes`
- Configure email theme in Keycloak admin.
- Register a new user in consent UI.
- Check Mailhog for your "verify your email" email.
